crates/octez/src/async/bootstrap.rs

```diff
@@ -5,57 +5,88 @@ use serde::de::{Error, Visitor};
 use serde::ser::SerializeSeq;
 use serde::{Deserialize, Serialize};
 use serde_json::Value;
-use tezos_crypto_rs::hash::ContractKt1Hash;
+use tezos_crypto_rs::hash::{ContractKt1Hash, SmartRollupHash};
 
 #[derive(Clone, PartialEq, Eq, Debug)]
 pub struct BootstrapAccount {
     public_key: PublicKey,
     amount_mutez: u64,
 }
 
-impl TryFrom<&Value> for BootstrapAccount {
-    type Error = anyhow::Error;
+impl Serialize for BootstrapAccount {
```

Comment:
Could you add a comment on how bootstrap accounts are represented in the params file?

Reply:
Done